src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php

```diff
@@ -173,10 +174,21 @@ protected function file(\SplFileInfo|string $file, string $fileName = null, stri
     protected function addFlash(string $type, mixed $message): void
     {
         try {
-            $this->container->get('request_stack')->getSession()->getFlashBag()->add($type, $message);
+            $session = $this->container->get('request_stack')->getSession();
         } catch (SessionNotFoundException $e) {
             throw new \LogicException('You cannot use the addFlash method if sessions are disabled. Enable them in "config/packages/framework.yaml".', 0, $e);
         }
+
+        if (!$session instanceof FlashBagAwareSessionInterface) {
+            trigger_deprecation(
+                'symfony/framework-bundle',
+                '6.2',
+                'Calling addFlash method when the session does not implements %s is deprecated.',
+                FlashBagAwareSessionInterface::class
+            );
```

Comment:
should be on one line to be consistent with the code base.

Reply:
Done